### PR TITLE
Nturinski/enhancement110

### PR DESCRIPTION
**Pull request by nturinski**
Tabling null checks for now-- all other tsconfig rules are enforced.

Did not realize that I did not have the tslint file yet :(

### CODE REVIEW

**Comment by nturinski** [comment]
Need to pend merge until Eric's PR is merged otherwise Travis CI build will break.

File: src/wizard.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ export class WizardBase {
 export interface WizardResult {
     status: WizardStatus;
     step: WizardStep;
-    error: Error;
+    error: Error | null;
```
Comment (EricJizbaMSFT) [inline src/wizard.ts]: undefined

File: src/webAppCreator.ts [hunk 2]
```diff
@@ -43,13 +43,13 @@ export class WebAppCreator extends WizardBase {
         return (<WebsiteStep>websiteStep).website;
     }
 
-    protected beforeExecute(step: WizardStep, stepIndex: number) {
+    protected beforeExecute(_step: WizardStep, stepIndex: number) {
```
Comment (EricJizbaMSFT) [inline src/webAppCreator.ts]: Can we just get rid of beforeExecute completely? This is the only place it's used and the output already has enough info:
```
Start creating new Web App...
The new Web App will be created in subscription "" ().
Existing resource group "" will be used.
Existing App Service Plan "" will be used.
Creating new Web App ""...
Web App "" created: 
```

Specifically, the 1st and 5th lines seem redundant to me

Comment (nturinski) [reply src/webAppCreator.ts]: @whiskeyjay I have no opinion on this-- Jay?

**Pull request merged**